lib/detail_page.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+
+import 'api/book.dart';
+
+/// Detail Page that displays information on the book
+class DetailPage extends StatelessWidget {
+  /// Creates a Detail Page and ensures book is not null
+  const DetailPage({@required this.book})
+      : assert(book != null, 'book cannot be null');
+
+  /// The book being displayed
+  final Book book;
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
+      appBar: AppBar(title: Text(book.title)),
+      body: ListView(
+        children: <Widget>[
```

Comment:
You don't need the `<Widget>` type here. Also to fix the alignment issue take a look at [SizedBox](https://api.flutter.dev/flutter/widgets/SizedBox-class.html).